recipes/xsimd/all/conanfile.py

```diff
@@ -46,7 +46,7 @@ def _compilers_minimum_version(self):
 
     def requirements(self):
         if self.options.xtl_complex:
-            self.requires("xtl/0.7.5")
+            self.requires("xtl/0.7.6")
```

Comment:
We'll need to sync this with the rest of the xtensor/xproperty packages if we don't want to generate conflicts. Could you open the corresponding ones?
In fact, checking into the upstream repository, we can probably use a version range here for the patch version at least for now, something like `xtl/[>=0.7 <0.8]`

Reply:
exception should be added to https://github.com/conan-io/conan-center-index/blob/b897f57ed86c971d915dc85a74f61d485c22e1ad/docs/adding_packages/dependencies.md?plain=1#L183

Reply:
yes, no worries about that, that's a document we keep updating when necessary :)